.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,38 @@
+name: try-runtime Workflow
+
+on: [pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        runtime: [ peregrine, spiritnet ]
+    runs-on: paritytech/ci-unified:bullseye-1.70.0
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Build
+        run: cargo build --release -p ${{ matrix.runtime }}-runtime --features try-runtime
+
+      - name: Run tests
+        uses: NodleCode/action-try-runtime@v0.5.1
```

Comment:
Do we want to bind to a third-party for this? Wouldn't we be able to cache this as well if we pull it from Parity GH releases as we did in the old version?

Reply:
Done